Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterUserMention.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Toolkit.Uwp.Services.Twitter
+{
+    /// <summary>
+    /// Twitter user object containing user mention indices.
+    /// </summary>
+    public class TwitterUserMention : TwitterUser
+    {
+        /// <summary>
+        /// Gets or sets the start and end position of the user mention
+        /// </summary>
+        [JsonProperty("indices")]
+        public int[] Indices { get; set; }
+
+        ///// <summary>
```

Comment:
Remove these if not used

Reply:
These are used for every single user mention in a tweet.

Reply:
Then why are they all commented out?